dev/com.ibm.ws.jaxb_fat/publish/servers/jaxb_optional_feature_toleration_fat/bootstrap.properties

```diff
@@ -0,0 +1,15 @@
+###############################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+com.ibm.ws.logging.trace.specification=*=info
+# This server is exempt from j2sec because it only tests third-party/JDK usage of JAX-B
+# The only Liberty code involved is trivial server start and servlet GET logic
+websphere.java.security.exempt=true
```

Comment:
this line can be removed too

Reply:
websphere.java.security.exempt=true line is removed